dascore/utils/tiles.py

```
@@ -65,14 +65,25 @@ def __post_init__(self):
-        return self.stride
+        """
+        Zeros added at each end of every axis, in whole strides.
+
+        Enough that every sample of the array is reached by every tile which
+        would reach it in an unbounded grid: one stride when tiles overlap
+        by no more than half, more when they overlap deeper, which a
+        synthesis window computed as a dual relies on.
+        """
+        return tuple(
+            max(step, math.ceil((z - step) / step) * step)
+            for z, step in zip(self.size, self.stride)
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)